app/views/unattended/partition_tables_templates/kickstart_default_encrypted.erb

```diff
@@ -0,0 +1,18 @@
+<%#
+kind: ptable
+name: Kickstart default encrypted
+model: Ptable
+oses:
+- AlmaLinux
+- RedHat
+- Rocky
+-%>
+<% if host_param('driverdisk_source') -%>
+driverdisk --source=<%= host_param('driverdisk_source') %>
+<% end -%>
+<% if host_param('ignoredisk_options') -%>
+ignoredisk <%= host_param('ignoredisk_options') %>
+<% end -%>
+zerombr
+clearpart --all --initlabel
+autopart --encrypted --passphrase="<%= host_param('disk_enc_passphrase', 'linux') %>" <%= host_param('autopart_options') %>
```

Comment:
this implies the default password is `linux` if none is given. Is this documented? Would it make sense to not encrypt if no passphrase is given?

Reply:
Documentation is work-in-progress here: https://github.com/theforeman/foreman-documentation/pull/2871

> Would it make sense to not encrypt if no passphrase is given?
This would require to always provide a intermediate passphrase via host parameter along with Tang server addresses. 
It's more obviously to the user what he can expect from the partition table when there are "encryption" and "non encryption" templates. But I am dispassionate on this topic.